.gitlab-ci.yml

```diff
@@ -100,13 +101,52 @@ build_tw_image:
         - src/python/TaskWorker/__init__.py
       policy: pull
 
+build_monit_image:
+  rules:
+    - !reference [.default_rules, default]
+    - !reference [.default_rules, release]
+  stage: build_monit
+  # needs: ["build_tw_image"]
```

Comment:
Any reason to comment it out?

Put this task inside `build_docker` and set `needs`  to `build_tw_image` look neater (also apply to `tag_monit_latest` job too).

![Screenshot from 2024-06-18 13-35-58](https://github.com/dmwm/CRABServer/assets/2346664/ef118e41-2ca6-43a0-8f1c-25bdfb806b2b)

src: https://github.com/novicecpp/CRABServer/blob/61e44eba686b0119faa72f5bd080dfe979e2d2a9/.gitlab-ci.yml#L109-L111

Just my suggestion, I leave the decision to you. I plan to refactor the pipeline later anyway (at least do some parent-child pipeline and/or generate it on-the-fly to use it as child pipeline, need to start migrating CV/CCV first).

Reply:
agreed, for some reason it did not work when i tried the first time, now it looks good :) https://gitlab.cern.ch/crab3/CRABServer/-/pipelines/7591546